types/react-simple-oauth2-login/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for react-simple-oauth2-login 0.5
+// Project: https://github.com/bhubr/react-simple-oauth2-login#readme
+// Definitions by: Kostas Antonopoulos <https://github.com/kostassite>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+export interface OAuth2LoginProps {
+    id?: string;
+    authorizationUrl: string;
+    clientId: string;
+    redirectUri: string;
+    responseType: 'code' | 'token';
+    onSuccess?: (data: Record<string, any>) => void;
+    onFailure?: (err: Error) => void;
```

Comment:
aren't those required?
https://github.com/bhubr/react-simple-oauth2-login/blob/master/src/OAuth2Login.jsx#L141-L142
```suggestion
    onSuccess: (data: Record<string, any>) => void;
    onFailure: (err: Error) => void;
```